resources/user-agents/browsers/chrome/chrome-android-43-on.json

```diff
@@ -64,6 +64,18 @@
             "Android_6_0"
           ]
         },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) Chrome/#MAJORVER#.#MINORVER#*Safari/*",
```

Comment:
This should be moved up above the Redmi one above it (but after the one that starts with "Android_8_1").

Reply:
done